couchable/testing/test_couchable.py

```diff
@@ -947,7 +954,7 @@ def test_simpleAttachments(self):
         c = SimpleDoc(name='CCC', attach=SimpleAttachment(c=1, cc=2), bb=b)
         a = SimpleDoc(name='AAA', attach=SimpleAttachment(a=1, aa=2), bb=b,
                 dt=datetime.datetime.now(),
-                td=datetime.timedelta(seconds=1.5),
+                td=datetime.timedelta(seconds=2.5),
```

Comment:
This needs to connect to a local CouchDB server. For my testing I port forwarded a dev host CouchDB port to my local port, but I needed to increase the timeout value for it to work. I could revert that if it should stay this way.